Tutorials/Notebooks/netket-VMC.ipynb

```diff
@@ -0,0 +1,426 @@
+      "text/plain": [
+       "<Figure size 432x288 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "fig, ax1 = plt.subplots()\n",
+    "ax1.plot(iters, energy, color='blue', label='Energy')\n",
+    "ax1.set_ylabel('Energy')\n",
+    "ax1.set_xlabel('Iteration')\n",
+    "ax2 = ax1.twinx() \n",
+    "ax2.plot(iters, np.array(neel)/(L*(L+2)), color='green', label='Neel')\n",
+    "ax2.set_ylabel('Neel')\n",
+    "ax1.legend(loc=2)\n",
+    "ax2.legend(loc=1)\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's also compute the average of those quantities (energy and neel order) over the last 50 iterations where the optimization seems to have connverged."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 91,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Neel order = 0.544(0.002)\n",
+      "Energy = -26.305(0.001)\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(r\"Neel order = {0:.3f}({1:.3f})\".format(np.mean(neel[-50:])/(L*(L+2)),\n",
+    "                                              np.std(np.array(neel[-50:])/(L*(L+2)))/np.sqrt(50)))\n",
+    "print(r\"Energy = {0:.3f}({1:.3f})\".format(np.mean(energy[-50:]), np.std(energy[-50:])/(np.sqrt(50))))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 6) Sanity Check: Exact Diagonalisation\n",
+    "\n",
+    "Now that we have obtained some results using VMC, it is a good time to check the quality of our results (at least for small system sizes). For this purpose, the ``netket`` library provides exact diagonalisation tools."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 92,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Exact ground state energy = -26.308\n"
+     ]
+    }
+   ],
+   "source": [
+    "res = nk.exact.lanczos_ed(op, first_n=1, compute_eigenvectors=True)\n",
```

Comment:
It is possible to add observables to ED results basically doing 
```python
# Compute exact neel order
print("g.s. neel order:", res.mean(neel_order, 0))
```

Reply:
ok i will add that